doc/glossary.rst

```diff
@@ -0,0 +1,1216 @@
+        TODO
+
+    estimator
+        TODO
+
+        The core functionality of some estimators may also be available as a
+        :term:`function`.
+
+    feature extractor
+    feature extractors
+        A :term:`transformer` which takes input where each sample is not
+        represented as an :term:`array-like` object of fixed length, and
+        produces an `array-like` object of :term:`features` for each sample
+        (and thus a 2-dimensional array-like for a set of samples).  In other
+        words, it (lossily) maps a non-rectangular data representation into
+        :term:`rectangular` data. Feature extractors should define
+        :term:`get_feature_names`.
+
+    meta-estimator
+    meta-estimators
+        TODO
+
+        Mention duck typing. Mention that duck typing of methods only works
+        after fitting. Mention lenient validation. ?Mention sample props.
+        Mention clone.
+
+    outlier detector
+        TODO
+
+    predictor
+        An estimator which provides :term:`predict`.  This encompasses
+        :term:`classifier`, :term:`regressor`, :term:`outlier detector` and
+        sometimes :term:`clusterer` (although the latter only provide
+        :term:`fit_predict` when they are not :term:`inductive`).
+
+    regressor
+        TODO
+        Mention :func:`~base.is_regressor`.
+
+    transformer
+        An estimator supporting :term:`fit` and :term:`transform` and/or
+        :term:`fit_transform`.  A purely :term:`transductive` transformer,
+        such as :class:`manifold.TSNE`, may not implement ``transform``.
+
+    vectorizer
+        See :term:`feature extractor`.
+
+    ``Xt``
+        Shorthand for transformed ``X``.
+
+There are further APIs specifically related to a small family of estimators,
+such as:
+
+* :class:`neighbors.DistanceMetric`
+* :class:`gaussian_process.kernels.Kernel`
+* ``tree.Criterion``
+* :term:`scorer`
+* :term:`CV splitter`
+
+.. _glossary_target_types:
+
+Target Types
+============
+
+.. glossary::
+
+    binary
+        A classification problem consisting of two classes.  A binary target
+        may represented as for a :term:`multiclass` problem but with only two
+        labels.  A binary decision function is represented as a 1d array.
+
+        Semantically, one class is often considered the "positive" class.
+        Unless otherwise specified (e.g. using :term:`pos_label` in
+        :term:`evaluation metrics`), we consider the class label with the
+        greater value as the positive class: of labels [0, 1], 1 is the
+        positive class; of [1, 2], 2 is the positive class; of ['no', 'yes'],
+        'yes' is the positive class; of ['no', 'YES'], 'no' is the positive
+        class.  This affects the output of :term:`decision_function`, for
+        instance.
+
+        Note that a dataset sampled from a multiclass ``y`` or a continuous
+        ``y`` may appear to be binary.
+
+        :func:`~utils.multiclass.type_of_target` will return 'binary' for
```

Comment:
the link [did not rendered](https://15591-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/glossary.html#term-binary), probably because of the `~`

Reply:
No, because it's not in `doc/modules/classes.rst`

Reply:
fair enough